shin-core/src/format/scenario/info.rs

```diff
@@ -6,16 +6,26 @@ use binrw::file_ptr::FilePtrArgs;
 use binrw::{BinRead, BinResult, BinWrite, Endian, FilePtr32};
 use std::io::{Read, Seek};
 
+/// Represents a mask, a black and white image specifying a transition between two screens.
```

Comment:
For `MaskInfoItem` and other asset references:

I... Don't like how this is worded

The `MaskInfoItem` doesn't _really_ represent a mask, it's merely a reference to it. This is better off in docs for `shin_core::format::mask::MaskTexture` and/or the struct in shin (when it will have masks). Probably not idea to touch other docs in this PR, just a thing to document later

Also, maybe it's worth adding a doc-link for `shin_core` struct representing the asset format and a shin struct representing the loaded asset (like `shin::asset::picture::Picture`)

Reply:
That's fair; I reworded stuff a bit such that it says more “references” instead of “represents”.

As for the doc links to `shin` itself: my thought was that `shin_core` would be useful on its own for people who want to make their own shin-related projects, so I avoided direct references to other parts of the engine. But if you think it would be useful regardless I can still add some links.

Reply:
Well, adding a link to `shin_core`'s format impl might still be useful.

As for the `shin` itself...  I am fine with no link being present here. Now that I think about it, I am not sure you can make such a link in rustdoc :thinking:.